README.markdown

```diff
@@ -423,18 +424,18 @@ It is also extremely appreciated to use comments to separate the [Class Attribut
 
 **Preferred:**
 ```swift
-class MyViewController                      : UIViewController {
+class MyViewController                           : UIViewController {
 
     // MARK: - Outlets
 
-    @IBOutlet var lettersGameButton         : UIButton?
-    @IBOutlet var headBodyLegsGameButton    : UIButton?
-    @IBOutlet var colorMatcherGameButton    : UIButton?
+    @IBOutlet weak var lettersGameButton         : UIButton?
+    @IBOutlet weak var headBodyLegsGameButton    : UIButton?
+    @IBOutlet weak var colorMatcherGameButton    : UIButton?
 
```

Comment:
I agree we should use that, but could you please double check if we HAVE to use it ? and what about with/without optionals ?